src/Models/PlayerCardModel.php

```diff
@@ -53,22 +53,23 @@ public function getPlayerCardById(int $id): PlayerCard|false
         return $Player;
     }
 
-    public function getAllCards($DeleteStatus): array
+    public function getAllCards(int $DeleteStatus, string $PositionId = ''): array
     {
         $query = $this->db->prepare(
             "SELECT `PremierLeagueCards`.`id`, 
-            `PremierLeagueCards`.`PlayerName`, 
-            `PremierLeagueCards`.`Club`, 
-            `Positions`.`PositionName`, 
-            `PremierLeagueCards`.`Defence`, 
-            `PremierLeagueCards`.`Control`, 
-            `PremierLeagueCards`.`Attack`, 
-            `PremierLeagueCards`.`Total`,
-            `PremierLeagueCards`.`Deleted` 
-            FROM `PremierLeagueCards` 
-            INNER JOIN `Positions` 
-            ON `PremierLeagueCards`.`Position` = `Positions`.`id`
-            WHERE `PremierLeagueCards`.`Deleted` = $DeleteStatus;"
+        `PremierLeagueCards`.`PlayerName`, 
+        `PremierLeagueCards`.`Club`, 
+        `Positions`.`PositionName`, 
+        `PremierLeagueCards`.`Defence`, 
+        `PremierLeagueCards`.`Control`, 
+        `PremierLeagueCards`.`Attack`, 
+        `PremierLeagueCards`.`Total`,
+        `PremierLeagueCards`.`Deleted` 
+        FROM `PremierLeagueCards` 
+        INNER JOIN `Positions` 
+        ON `PremierLeagueCards`.`Position` = `Positions`.`id`
+        WHERE `PremierLeagueCards`.`Deleted` = $DeleteStatus" .
+                ($PositionId ? " AND `Positions`.`id` = $PositionId" : "")
```

Comment:
You need to use parameter binding for all of thee variables that are part of the query. This is currently vulnerable to sql injection